karma.conf.js

```diff
@@ -5,13 +5,26 @@ module.exports = config => {
       'jasmine'
     ],
     files: [
-      'test/**/*.js'
+      'node_modules/jasmine-promises/dist/jasmine-promises.js',
+      'test/*.js'
+      // 'src/*.js'
+      // 'test/pronunciationAnalysisSpec.js',
+      // 'test/speechRecordingSpec.js'
     ],
     browsers: [
       'PhantomJS'
     ],
+
+    reporters: ['progress', 'coverage'],
+
     preprocessors: {
-      'test/**/*.js': ['browserify']
+      'test/*.js': ['browserify'],
+      '**/*.js': ['coverage']
+    },
+
+    coverageReporter: {
+      type: 'html',
+      dir: 'coverage/'
```

Comment:
Coverage related changes are unrelated to this PR. Also, coverage has already been implemented in the latest master. (See #6)